webapp/src/Components/Player/Overview/PlayStyle/PlayerPlayStyle.tsx

```diff
@@ -73,7 +79,7 @@ class PlayerPlayStyleComponent extends React.PureComponent<Props, State> {
     }
 
     private readonly getPlayStyles = (): Promise<void> => {
-        return getPlayStyle(this.props.player.id)
+        return getPlayStyle(this.props.player.id, undefined, this.props.playlist)
```

Comment:
I'm not sure how to skip the rank parameter and just use playlist, so I had to use `undefined` here.

Reply:
This is standard: https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters